packages/main/src/Form.ts

```diff
@@ -44,6 +44,7 @@ type GroupItemsInfo = {
 	groupItem: IFormItem,
 	classes: string,
 	items: Array<ItemsInfo>,
+	ariaLabelledBy: string | undefined
```

Comment:
probably this type conflicts too with the definition of aria-labelledby native attribute.

Reply:
done